tests/spec/util/pagination-utils.spec.ts

```diff
@@ -5,41 +5,17 @@ import { loadMoreDataWhenScrolled } from '../../../react-jhipster';
 describe('loadMoreDataWhenScrolled', () => {
   const setLinks = (first, last, prev) => ({ first, last, prev });
   const state = { entities: [{ 1: 'fake1' }, { 2: 'fake2' }, { 3: 'fake3' }] };
-  const payload = { data: [{ 4: 'fake4' }, { 5: 'fake5' }] };
-  const otherPayload = { data: [{ 7: 'fake7' }, { 8: 'fake8' }, { 9: 'fake9' }] };
-  const itemsPerPage = 2;
+  const payload = { data: [{ 1: 'fake4' }, { 2: 'fake2' }, { 3: 'fake3' }] };
```

Comment:
shouldn't state and payload have same keys, else the list will not be merged and you might get same stuff twice?